docker/docker-entrypoint.sh

```diff
@@ -0,0 +1,51 @@
+#!/bin/bash
+# Copyright 2018 Evgeny Golyshev <eugulixes@gmail.com>
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -e
+
+set -x
+
+export ADDR=${ADDR:="127.0.0.1"}
+
+export PORT=${PORT:=8006}
+
+export HOST=${HOST:="http://$ADDR:$PORT"}
+
+export USERNAME=${USERNAME:=""}
+
+export PASSWORD=${PASSWORD:=""}
+
+export WAIT=${WAIT:='80'}
+
+export PUGS_LIMIT=${PUGS_LIMIT:=5}
+
+export PYTHON=${PYTHON:="python3"}
+
+BRANCH=${BRANCH:=docker-tests}
```

Comment:
The list should be sorted by alphabet. Also, one type of brackets should be used and all default args should be strings.

Reply:
The list cannot be sorted in alphabetical order because some of the params depend on others.

Reply:
Ok.